chapters/10-community-involvement.Rmd

```diff
@@ -9,7 +9,28 @@
 - NEWS file
 - reprex
 
+## Outline of action steps
 
+This chapter is very description-heavy with minimal code.
+
+- Creating a welcoming community with welcoming materials
+<!-- I grouped reprex and NEWS with the README because they seem related to the general welcoming presentation of a project, but they could go elsewhere. -->
```

Comment:
I initially waffled about where to place NEWS and reprex, but I think the way you've framed it here is great. I am wondering about the interactions between this README explanation and other places where we discuss documentation?

Reply:
True, right now README is first introduced in [chapter 8](https://github.com/merely-useful/r-rse/blob/main/chapters/08-documentation.Rmd) I believe. Maybe in this chapter the focus should be more about how to make an existing README welcoming, whereas in chapter 8 it's about putting in a README right from the beginning.

Reply:
Definitely agree!